app/src/test/java/com/codesoom/assignment/application/UserServiceTest.java

```diff
@@ -0,0 +1,248 @@
+                then(userRepository).should(times(1))
+                        .save(any(User.class));
+            }
+        }
+    }
+
+    @Nested
+    @DisplayName("updateUser 메소드는")
+    class Describe_Update_User {
+        @Nested
+        @DisplayName("목록에 유저가 있다면")
+        class Context_Valid_Update_User {
+            private Long id = 1L;
+            private UserData userData;
+            private Optional<User> foundUser;
+
+            @BeforeEach
+            void setUpContextValidUpdateUser() {
+                userData = setUserData(id);
+                foundUser = setUserByUserData(userData);
+                given(userRepository.findUserById(eq(id))).willReturn(foundUser);
+                given(userRepository.save(any(User.class)))
+                        .willReturn(foundUser.get());
+            }
+
+            @Test
+            @DisplayName("유저 정보를 수정한다.")
+            void valid_update_user() {
+                UserData updateUserData = userService.updateUser(id, userData);
+                assertAll(
+                        () -> {
+                            Assertions.assertThat(updateUserData)
+                                    .extracting("id")
+                                    .isEqualTo(id);
+                        },
+                        () -> {
+                            Assertions.assertThat(updateUserData)
+                                    .extracting("email")
+                                    .isEqualTo(userData.getEmail());
+                        },
+                        () -> {
+                            Assertions.assertThat(updateUserData)
+                                    .extracting("name")
+                                    .isEqualTo(userData.getName());
+                        },
+                        () -> {
+                            Assertions.assertThat(updateUserData)
+                                    .extracting("password")
+                                    .isEqualTo(userData.getPassword());
+                        }
+                );
+                then(userRepository).should(times(1))
+                        .findUserById(eq(id));
+                then(userRepository).should(times(1))
+                        .save(any(User.class));
+            }
+        }
+
+        @Nested
+        @DisplayName("목록에 유저가 없다면")
+        class Context_Invalid_Update_User {
+            private Long id = 100L;
+            private UserData userData;
+            private String errorMessage = userNotFoundMessage + id;
+
+            @BeforeEach
+            void setUpInvalidUpdateUser() {
+                userData = setUserData(id);
+                given(userRepository.findUserById(eq(id)))
+                        .willThrow(new UserNotFoundException(id));
+
+            }
+
+            @Test
+            @DisplayName("UserNotFoundException이 발생한다.")
+            void invalid_update_user_exception() {
+                Assertions.assertThatThrownBy(
```

Comment:
바로 아래의 코드를 읽어봅시다. `assertThatThrownBy`라고 되어 있지요?
```suggestion
            @DisplayName("UserNotFoundException을 던진다")
            void invalid_update_user_exception() {
                Assertions.assertThatThrownBy(
```

Reply:
네 던진다라는 표현으로 변경했습니다.